tests/context.py

```
@@ -116,10 +116,9 @@
 try:
 	if 'packaging' not in sys.modules:
 		import packaging
-		from packaging import version
 	else:  # pragma: no branch
 		packaging = sys.modules["packaging"]
-		from packaging import version
+	from packaging import version
 except ImportError as err:  # pragma: no branch
 	raise ModuleNotFoundError("[CWE-440] packaging.version Failed to import.") from err
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)